core/src/main.cpp

```diff
@@ -161,7 +161,7 @@ constexpr auto default_jsonschema() -> std::string_view
 	// clang-format on
 	return R"({{
     "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "$id": "https://schemas.irods.org/irods-http-api/0.4.0/schema.json",
+    "$id": "https://schemas.irods.org/irods-http-api/config.json",
```

Comment:
neither of these URLs (before or after) are real...

and not sure we want to introduce new dependencies on schemas.irods.org... it is still alive only because early 4.2 pointed there to resolve things I believe...

Reply:
Correct. The URL is only an identifier. It means nothing to the application. iRODS 5 does the same thing.

This aligns with the issue about removing relative paths from the schema files. See https://github.com/irods/irods/issues/6283.

Reply:
okay, so... do we have a suitable suggestion for replacement here?

is the `$id` required?   and what can satisfy it?

Reply:
Replacement? Can you elaborate?

The requirements of `$id` are already satisfied. The only requirement is that it be unique. It does not need to resolve to anything.

Reply:
unique... so it doesn't even need to be a url?   

suggestion is... no.

https://json-schema.org/understanding-json-schema/basics#declaring-a-unique-identifier

Reply:
the latest draft...
https://json-schema.org/draft/2020-12/draft-bhutton-json-schema-01#section-8.2.1

> Note that this URI is an identifier and not necessarily a network locator. In the case of a network-addressable URL, a schema need not be downloadable from its canonical URI.

so we're good with
```
https://schemas.irods.org/irods-http-api/config.json 
```

marking as resolved.